vllm/compilation/backends.py

```diff
@@ -199,6 +219,7 @@ def wrap_inductor(graph: fx.GraphModule,
                 "Inductor cache lookup failed. Please remove"
                 f"the cache file {cache_data.cache_file_path} and try again."  # noqa
             )
+            inductor_artifact.file_path = inductor_compiled_graph.current_callable.__code__.co_filename  # noqa
```

Comment:
Why do we only update the file_path of `graph_index` 0? Do we only support fullgraph mode of `torch.compile`?

Reply:
seems to be misunderstanding. `if graph_index == 0:` is only for logging, this line is executed for every piece of graph.